component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -158,7 +158,9 @@
                                 "This should be the absolute path of the file that going to be created after moving " +
                                 "is done.\n" +
                                 "This uri MUST have the respective protocol specified.\n",
-                        type = {DataType.STRING}
+                        type = {DataType.STRING},
+                        optional = true,
+                        defaultValue = "file:/var/tmp"
```

Comment:
Shall we change this to defaultValue = "<Empty_String>" ? since that's how we have done in the other extensions.